ci-operator/config/code-ready/crc/code-ready-crc-master.yaml

```diff
@@ -35,7 +35,7 @@ tests:
   as: e2e-crc
   steps:
     cluster_profile: gcp
-    workflow: crc-e2e
+    workflow: code-ready-crc-e2e
```

Comment:
I thought the `code-ready` name on github was not great, and that `codeready` would have been preferred?

Reply:
in general maybe not a 'good name' as it is a brand group

Reply:
@gbraad @cfergeau As of now this is the org name (code-ready) and I don't want to mess around it until we do make some changes github org side.

Reply:
After further discussion/tests, the directory hierarchy must match the github hierarchy, so we can't change `code-ready/crc` in paths. The question which remains is whether to fix the non-path naming to use `codeready`, or to be consistent. I agree it's probably less confusing to use `code-ready` for these too at the moment.